test/unit/calculators/state_pension_age_calculator_test.rb

```diff
@@ -60,6 +60,78 @@ class StatePensionAgeCalculatorTest < ActiveSupport::TestCase
           assert_equal @calculator.state_pension_age, '60 years'
         end
       end
+
+      context 'given variable changes to state pension age between 66 - 67 years for births between 6 Apr 1960 - 5 Mar 1961' do
+        [
+          {
+            birth_date: Date.parse('6 Apr 1960'),
+            pension_date: Date.parse('6 May 2026'),
+            expected_age: '66 years, 1 month'
+          },
+          {
+            birth_date: Date.parse('20 May 1960'),
+            pension_date: Date.parse('20 Jul 2026'),
+            expected_age: '66 years, 2 months'
+          },
+          {
+            birth_date: Date.parse('1 Jul 1960'),
+            pension_date: Date.parse('1 Oct 2026'),
+            expected_age: '66 years, 3 months'
+          },
+          {
+            birth_date: Date.parse('8 Dec 1960'),
+            pension_date: Date.parse('8 Sep 2027'),
+            expected_age: '66 years, 9 months'
+          },
+          {
+            birth_date: Date.parse('28 Feb 1961'),
+            pension_date: Date.parse('28 Jan 2028'),
+            expected_age: '66 years, 11 months'
+          },
+        ].each do |test_case|
+          should "someone born on #{test_case[:birth_date]} and getting their pension on #{test_case[:pension_date]} should have a pension age of #{test_case[:age]}" do
+            @calculator.stubs(:dob).returns(test_case[:birth_date])
+            @calculator.stubs(:state_pension_date).returns(test_case[:pension_date])
+            assert_equal @calculator.state_pension_age, test_case[:expected_age]
```

Comment:
We've got our arguments in the wrong order here. This will give us confusing failure messages.

It should be `assert_equal(expected, actual)` not `assert_equal(actual, expected)`

See https://apidock.com/ruby/Test/Unit/Assertions/assert_equal

Reply:
Noted